messaging/tests/test_messaging.py

```diff
@@ -175,7 +175,8 @@ def test_recv_sock(self):
     time.sleep(0.01)
     recvd = messaging.recv_sock(sub_sock)
     self.assertIsInstance(recvd, capnp._DynamicStructReader)
-    assert_carstate(msg.carState, recvd.carState)
+    # https://github.com/python/mypy/issues/13038
+    assert_carstate(msg.carState, recvd.carState)  # type: ignore[union-attr]
```

Comment:
mypy doesn't look at unittest's asserts for type narrowing. we can do `if recvd is not None:`, but that is a similar solution to just ignoring it